hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxy.java

```diff
@@ -403,8 +401,7 @@ protected void updateCacheListenerConfigOnOtherNodes(CacheEntryListenerConfigura
                 final ClientMessage request =
                         CacheListenerRegistrationCodec.encodeRequest(nameWithPrefix, configData, isRegister, address);
                 final ClientInvocation invocation = new ClientInvocation(client, request, address);
-                final Future future = invocation.invoke();
-                futures.add(future);
+                invocation.invoke();
```

Comment:
@Donnerbart yes, it is not used at the moment and I think it is safe to delete because it is about updating cache config not the actual listener registration which is sync. So I think, no need to wait update cache configs as sync, async should be OK.